services/efs-guardian/src/simcore_service_efs_guardian/services/process_messages.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from fastapi import FastAPI
+from models_library.rabbitmq_messages import DynamicServiceRunningMessage
+from pydantic import parse_raw_as
+from simcore_service_efs_guardian.services.modules.redis import get_redis_lock_client
+
+from ..core.settings import get_application_settings
+from ..services.efs_manager import EfsManager
+
+_logger = logging.getLogger(__name__)
+
+
+async def process_dynamic_service_running_message(app: FastAPI, data: bytes) -> bool:
```

Comment:
This is a message_handler that you provide in the rabbit_client.subscribe() method. Returning True means everything went well and you are telling rabbit mq that it can remove the message from the queue.